codeowners/src/codeowners.rs

```diff
@@ -3,10 +3,12 @@ use std::{
     path::{Path, PathBuf},
 };
 
-use codeowners::{FromReader, GitHubOwners, GitLabOwners};
+use crate::github::GitHubOwners;
+use crate::gitlab::GitLabOwners;
+use crate::traits::FromReader;
```

Comment:
nit, optional: we can group these imports a little more nicely for consistency
```suggestion
use crate::{github::GitHubOwners, gitlab::GitLabOwners, traits::FromReader};
```